src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CreateReferralController.java

```diff
@@ -163,6 +163,9 @@ public ResponseEntity<CCDCallbackResponse> aboutToSubmitReferralDetails(
         }
 
         CaseData caseData = ccdRequest.getCaseDetails().getCaseData();
+        if ("Party not responded/compiled".equals(caseData.getReferralSubject())) {
+            caseData.setReferralSubject("Party not responded/complied");
```

Comment:
I have a feeling this will require a new config entry

Reply:
Can we hide that (duplicated) entry from the subjects list?

Reply:
Can't hide fixed lists. I think as long as the label is showing the right spelling, it should be ok

Reply:
As spoken to Harpreet, we would need this change for frontend to show correct value in the tab